File: src/net/eduard/api/command/map/MapCommand.java

package net.eduard.api.command.map;

import net.eduard.api.setup.manager.CommandManager;

public class MapCommand extends CommandManager {

	public MapCommand() {
		super("map");
		register(new MapCopyCommand());
		register(new MapPasteCommand());
		register(new MapPos1Command());
		register(new MapPos2Command());
		register(new MapLoadCommand());
		register(new MapSaveCommand());
		register(new MapWorldCommand());
		register(new MapHelpCommand());
	}

}

File: src/main/java/net/eduard/api/command/map/MapCommand.kt
package net.eduard.api.command.map

import net.eduard.api.lib.manager.CommandManager

class MapCommand : CommandManager("map") {
    init {
        register(MapCopyCommand())
        register(MapPasteCommand())
        register(MapPos1Command())
        register(MapPos2Command())
        register(MapLoadCommand())
        register(MapSaveCommand())
        register(MapSetCommand())
        register(MapListCommand())
        register(MapHelpCommand())
    }
}